src/test/java/seedu/task/commons/util/UrlUtilTest.java

```diff
@@ -1,14 +1,12 @@
 package seedu.task.commons.util;
 
```

Comment:
I think we can delete this test. As well as the Browser panel. Since we are not using it anymore. It will reduce lines of code